stonesoup/metricgenerator/ospametric.py

```diff
@@ -319,6 +350,8 @@ def compute_gospa_metric(self, measured_states, truth_states):
         if num_truth_states == 0:
             # When truth states are empty all measured states are false
             opt_cost = -1.0 * num_measured_states * dummy_cost
+            if self.alpha == 2:
+                gospa_metric['false'] = opt_cost
```

Comment:
This change was not part of my first patch, but came up as I added some tests to provide coverage of my original changes.

It seems reasonable that GOSPA should report tracks as false also in the case where there are 0 ground truth states, right?

I don't fully understand why we test for `self.alpha == 2`, but this is what is done everywhere else (see for instance [line 369](https://github.com/dstl/Stone-Soup/pull/1059/files#diff-0cd5f02bb81ea6e2a48b6ad697fdc78814cfbe9a041df8458670dbca164ab153R369)).